src/update_engine_ie9.jsx

```diff
@@ -64,8 +64,10 @@ export function update() {
  */
 export const requestAsyncUpdate = _.debounce(updateAfterDelay, 0);
 
-// update on window resize
-window.addEventListener('resize', _.debounce(requestAsyncUpdate, 16));
+if (typeof window !== 'undefined') {
```

Comment:
yeah good idea. better to export a function that gets called in index

On Mon, Jan 25, 2016 at 12:14 PM Leonardo Garcia Crespo <
notifications@github.com> wrote:

> In src/update_engine_ie9.jsx
> <https://github.com/byronmwong/react-flexbox-layout/pull/12#discussion_r50749102>
> :
>
> > @@ -64,8 +64,10 @@ export function update() {
> >   */
> >  export const requestAsyncUpdate = _.debounce(updateAfterDelay, 0);
> >
> > -// update on window resize
> > -window.addEventListener('resize', _.debounce(requestAsyncUpdate, 16));
> > +if (typeof window !== 'undefined') {
>
> It is also being required in all IE9 components. Probably it makes sense
> to add this listener in index? Or maybe export a function startListeners
> or similar that can be called on index if we're in a DOM and we're
> simulating flexbox.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/byronmwong/react-flexbox-layout/pull/12/files#r50749102>
> .
>